packages/tailwindcss/src/utilities.ts

```diff
@@ -1931,8 +1931,11 @@ export function createUtilities(theme: Theme) {
   staticUtility('break-all', [['word-break', 'break-all']])
   staticUtility('break-keep', [['word-break', 'break-keep']])
 
+  /**
+   * @css `border-radius`
+   */
   {
-    // border-radius
+    // Deprecated: `rounded` utilities
```

Comment:
I wish we could generate a `/* @deprecated */` comment next to the utility so that it shows in some editors as deprecated. But Lightning CSS sadly strips comments away (or their internal parser does).